src/centos/stream9/amd64/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN dnf install --setopt tsflags=nodocs --refresh -y \
         ncurses-devel \
         numactl-devel \
         openssl-devel \
+        pigz \
```

Comment:
Is pigz in the base repository for Centos Stream 9? I'd like to keep this image as similar as possible to how our partners at RedHat would build dotnet from source.

Reply:
Yes, it is available there. This only improves the speed up our tar usage when available; perf numbers are in the linked issue.

Reply:
https://github.com/dotnet/arcade/pull/15038#discussion_r1743480848

Reply:
Looks like CentOS found it in baseos repo:

<img width="1499" alt="image" src="https://github.com/user-attachments/assets/f177dd8c-2389-4a0c-8088-25d0920f23cc">

Reply:
cc @omajid FYI